recipes/conversation_view/attachment_carousel/attachment_carousel.vue

```diff
@@ -0,0 +1,263 @@
+<template>
+  <div
+    class="dt-attachment-carousel"
+    role="presentation"
+    @mouseenter="showCarouselArrows = true"
+    @mouseleave="showCarouselArrows = false"
+    @focusin="showCarouselArrows = true"
+    @focusout="showCarouselArrows = false"
+  >
+    <div
+      v-if="mediaList.length > 0"
+      ref="carousel"
+      class="dt-attachment-carousel--media-list"
+      :aria-label="attachmentAriaLabel"
+      @scroll="handleScroll"
+    >
+      <!-- Image list -->
+      <div
+        v-for="(mediaItem, index) in mediaList"
+        :key="`media-${index}`"
+        role="presentation"
+        class="d-ps-relative"
+        @focusin="closeButton(true, index)"
+        @focusout="closeButton(false, index)"
+        @mouseenter="closeButton(true, index)"
+        @mouseleave="closeButton(false, index)"
+      >
+        <dt-image-viewer
+          image-button-class="d-h64 d-w64 d-bar4 d-ba d-baw6 d-bc-subtle"
+          :image-src="mediaItem.path"
+          :image-alt="mediaItem.altText"
+          :close-aria-label="closeAriaLabel"
+          :aria-label="clickToOpenAriaLabel"
```

Comment:
Should you add the alt text in here so it says "click to open" and then a description of the image?

Reply:
`image-alt`  is passed onto the image viewer alt text. Is that what you are talking about?

Reply:
I was thinking something like.. ``:image-alt="`Click to open ${mediaItem.altText}`"``

But I realized this is an i18n issue so nevermind.